lib/app/connector.rb

```diff
@@ -51,8 +51,9 @@ def start_polling_jobs
         loop do
           polling_jobs
         rescue StandardError => e
-          Utility::Logger.error("Error happened during sync. error: #{e.message}")
+          Utility::ExceptionTracking.log_exception(e, 'Sync failed due to unexpected error.')
         ensure
+          Utility::Logger.info("Sleeping for #{POLL_IDLING} seconds")
```

Comment:
maybe not now, but we'll need a mechanism that stops the program if we have a failure at every call in the loop.
For instance 3 times the same exception in a row or some kind of threshold

Reply:
Yep, can do that in future: we have https://github.com/elastic/connectors/blob/main/lib/utility/monitor.rb is a class encapsulating this kind of logic

Reply:
I didn't realize that Monitor had made it into connectors. This makes me happy. :)